tinygrad/ops_ane.py

```diff
@@ -24,8 +21,8 @@ def compile_relu(ane, sz):
 
 class ReLU(Function):
   @staticmethod
-  def forward(ctx, input):
-    ret = ctx.ane.tensor(input.shape)
-    ctx.ane.run(compile_relu(ctx.ane, input.sz), input, ret)
+  def forward(ctx, _input):
+    ret = ctx.ane.tensor(_input.shape)
+    ctx.ane.run(compile_relu(ctx.ane, _input.sz), _input, ret)
```

Comment:
`input` is a reserved word in python, it's good to avoid overwriting -- even though you probably wont use input